internal/testplanet/planet.go

```diff
@@ -71,8 +71,9 @@ type Config struct {
 	StorageNodeCount int
 	UplinkCount      int
 
-	Identities  *Identities
-	Reconfigure Reconfigure
+	Identities         *Identities
+	Reconfigure        Reconfigure
+	UsePeerCAWhitelist bool
```

Comment:
I'm not sure whether it's necessary to add this. I think we can by default use ca whitelist. For testing disallowing connection we can either 
1. reconfigure a single storage node or satellite to use a different identity
2. create a bootstrap/satellite/storagenode Peer separately from testplanet and try to connect to testplanet using it.

Reply:
I didn't want to be presumptuous and/or deal with making it the default in this PR. Also, it's worth considering that this isn't going to be the default for non-alpha networks.

Reply:
I mean then we can always disable it later :)... I mean I'm not against this approach... just thinking it might be easier to handle it by making it default.

Reply:
I don't feel very strongly about it but I think it makes sense to do it this way for now. I think we should be testing both signed and unsigned networks.

Reply:
My thought process is that
1. when it works with signed, it will most likely work with unsigned;
2. since only few tests actually care about the actual signedness, then there's little benefit making it an option.

Reply:
i wasn't seeing how to "reconfigure" a `Peer`'s identity aside from what I was doing with mixed identites and by messing with  planet.go:171,479. The identity is being pulled from the pregenerated list that's passed in the config.

Reply:
Ah right, forgot there wasn't a hook for that. I feel like there's an elegant solution, but I'm not seeing it yet :)

Reply:
What if we pass into reconfigure funcs also a pointer to the identity.... ah... hmm

Reply:
in `testplanet` if we have `type SatelliteSetup struct { Identity *Identity; DB satellite.DB; Config *satellite.Config}` then... we remove the specific `NewSatelliteDB` and instead have a single reconfigure func that takes that `SatelliteSetup` as argument.

for reconfigure only `Config` is initialized... if identity/db is missing after reconfigure it will use the default sources...